config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id/items", to: "customer_items#create"
```

Comment:
I like this choice here with this path to the controller action. 
All routes look good and restful.